### PR TITLE
Fix OpenSCAP longrun tests

### PR DESCRIPTION
**Pull request by jameerpathan111**
This PR:
- fixes OpenSCAP longrun tests failing because of memory limitation of host.

See PRT job no. 257 for test results.

### CODE REVIEW

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_oscap_run_via_ansible_bz_1814988

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_oscap_run_via_ansible

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_oscap_run_via_ansible_bz_1814988

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_oscap_run_via_ansible

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

File: robottelo/hosts.py [hunk 1]
```diff
@@ -499,6 +501,29 @@ def configure_puppet(self, rhel_repo=None, proxy_hostname=None):
...
+        """Invoke a job on the host by passing name of the job-template.
+
+        :param job-template: name of job-templete to be used.
+        :return: None.
+
+        """
+        invocation_command = make_job_invocation(
+            {'job-template': job_template, 'search-query': f"name ~ {self.hostname}"}
+        )
+        result = JobInvocation.info({'id': invocation_command['id']})
```
Comment (mshriver) [inline robottelo/hosts.py:514]: These classes should be referenced through the `self.cli` namespace.

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_upload_to_satellite

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_upload_to_satellite

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_upload_to_satellite

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py -k test_positive_upload_to_satellite

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

File: robottelo/hosts.py (hunk 1 above)
Comment (jameerpathan111) [reply robottelo/hosts.py]: Jobinvocation is not working for me due to unknown reason.
Removing it from this pr and will raise a separate pr once
the issue is fixed. Reverting code to use old way of running openscap scans.

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Comment by jameerpathan111** [comment]
Test result:
```
11:46:26  + py.test tests/foreman/longrun/test_oscap.py --junit-xml=sat-results.xml -o junit_suite_name=sat-result
11:46:31  ============================= test session starts ==============================
11:46:31  collected 13 items / 6 deselected / 7 selected
13:21:21  tests/foreman/longrun/test_oscap.py .......                              [100%]
13:21:21  =========== 7 passed, 6 deselected, 4 warnings in 5680.54s (1:34:40) ===========
```

File: tests/foreman/longrun/test_oscap.py [hunk 2]
```diff
@@ -58,6 +58,8 @@
     'rhel7': gen_string('alpha'),
     'rhel6': gen_string('alpha'),
 }
+target_cores = 4
+target_memory = '16GiB'
```
[the remark this answers is not in the document]
Comment (jameerpathan111) [reply tests/foreman/longrun/test_oscap.py]: updated the pr.

**Review by devendra104 — APPROVED** [review]

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
@jameerpathan111 seeing two test cases fail in robottelo-runner against your latest commit. Your local test results were posted before the latest commit. Can you please investigate?

**Comment by jameerpathan111** [comment]
@mshriver I don't think tests failed because of the last commit. I will run the PRT job again to make sure.

**Comment by jameerpathan111** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/longrun/test_oscap.py

**Pull request merged**